src/test/java/com/epam/edumanagementsystem/admin/impl/SuperAdminServiceImplTest.java

```diff
@@ -0,0 +1,62 @@
+package com.epam.edumanagementsystem.admin.impl;
+
+import com.epam.edumanagementsystem.admin.model.entity.SuperAdmin;
+import com.epam.edumanagementsystem.admin.rest.repository.SuperAdminRepository;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.when;
+
+@SpringBootTest
+class SuperAdminServiceImplTest {
+
+    @MockBean
+    private SuperAdminRepository superAdminRepository;
+
+    @Autowired
+    private SuperAdminServiceImpl superAdminService;
+
+    @Test
+    void testFindByEmailResultIsEmptyWithWrongEmail() {
+        String email = "wrong@gmail.com";
+        when(superAdminRepository.findByEmail(email)).thenReturn(Optional.empty());
```

Comment:
If in the system we are returning some message to the user when it was not possible to find the user with specified email, we can assert that message. If not, just keep this as it is.